### PR TITLE
Added new intersteing features

### PR DESCRIPTION
**Pull request by franksalgado**
hi #1

### CODE REVIEW

File: src/pages/About/About.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import React from 'react'
 const About = () => {
   return (
     <Container>
-        About
+        <p>fuck you</p>
```
Comment (vudiep411) [inline src/pages/About/About.js]: please be appropriate :)

**Review by vudiep411 — COMMENTED** [review]
change this please

File: src/pages/About/About.js (hunk 1 above)
Comment (franksalgado) [reply src/pages/About/About.js]: changed it boss

**Review by vudiep411 — APPROVED** [review]
nice!

**Pull request merged**